### PR TITLE
Format and add a test

### PR DESCRIPTION
**Pull request by eed3si9n**
Currently this repo lacks validation beyond compilation. This improves the situation slightly by adding a "dummy-bridge" implementation, which is an actual bridge proposed in https://github.com/sbt/zinc/pull/712.

The test currently tests that the bridge is capable of building an in-memory VirtualFile.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,4 +10,4 @@ env:
   - ADOPTOPENJDK=8
   - ADOPTOPENJDK=11
 
-script: sbt -Dfile.encoding=UTF8 compilerInterface/mimaReportBinaryIssues headerCheck packageBin doc
+script: sbt -Dfile.encoding=UTF8 compilerInterface/headerCheck jvmfmtCheck dummyBridge/test doc
```
Comment (SethTisue) [inline .travis.yml:13]: `compilerInterface/mimaReportBinaryIssues` isn't necessary?

Comment (eed3si9n) [reply .travis.yml:13]: Not for the first version, which I am about to move to a new package.

Comment (SethTisue) [reply .travis.yml:13]: shall I open an issue to remind us to re-enable it?

Comment (eed3si9n) [reply .travis.yml:13]: I opened one - https://github.com/scala/compiler-interface/issues/12

**Pull request merged**